cypress/integration/logistics/create_new_tour_and_transportation_order_spec.js

```diff
@@ -0,0 +1,22 @@
+describe('create new tour and transportation order', function() {
+  before(function() {
+    cy.visit('/window/540331');
+  });
+
+  const testTour = 'TestTour';
```

Comment:
please add a timestamp to make the name unique and the test repeatable on the same DB